src/components/TransformWorkflow/FlowEditor/Components/LowerSectionTabs/PreviewPane.tsx

```diff
@@ -199,8 +199,9 @@ const PreviewPane = ({ height }: { height: number }) => {
           variant="contained"
           sx={{ padding: '5px' }}
           disabled={downloadInProgress}
+          data-testid="downloadbutton"
         >
-          {downloadInProgress ? (
+          {false ? (
```

Comment:
why?

Reply:
-My mistake, was testing something. REMOVED NOW. set again to downloadInProgess.